src/h5web/toolbar/controls/DomainSlider/BoundEditor.tsx

```diff
@@ -83,16 +83,19 @@ function BoundEditor(props: Props): ReactElement {
         className={styles.actionBtn}
         type="submit"
         disabled={!isEditing}
-        aria-label="Apply"
+        aria-label={`Apply ${bound}`}
```

Comment:
This helps me find the correct apply/cancel buttons without resorting to `getAll` + array access.